### ISSUE TITLE
Bad usage of wsdl2java on windows

### ISSUE THREAD

**Issue by sortega** [ebiznext/sbt-cxf-wsdl2java#3]
Hello,

When using the plugin on windows I get the following error:

```
 WSDLToJava Error: org.apache.cxf.tools.common.toolspec.parser.BadUsageException: Unexpected option: -verbose
```

Looks like that option is not supported on that platform. Can you make verbosity configurable as a workaround?

**Comment by sortega** [comment]
Closing as this was ignored for too long

**Comment by fupelaqu** [comment]
has been fixed with #8
